lesson 6/6_1.js

```diff
@@ -0,0 +1,13 @@
+function addThemAll(...arguments) {
+    let sum = 0;
+    for (const num of arguments) {  
+        sum = num + sum
+    }
```

Comment:
Please, do not name arguments as `arguments` it's confusing a little bit :) I't better to name it like `args` or `numbers`.